sdk/core/System.Net.ClientModel/tests/client/OpenAIClient/Choice.Serialization.cs

```diff
@@ -13,8 +15,11 @@ namespace OpenAI;
 
 public partial class Choice
     {
-        internal static Choice DeserializeChoice(JsonElement element)
+        internal static Choice DeserializeChoice(JsonElement element, ModelReaderWriterOptions options)
         {
+            bool wire = options.Format == ModelReaderWriterFormat.Wire;
+            if (options.Format != ModelReaderWriterFormat.Json) throw new ArgumentOutOfRangeException(nameof(options.Format));
```

Comment:
*confused* - won't this make it so we can never deserialize if options.Format == Wire?  Tell me what I'm missing, I think I must just be misunderstanding.

Reply:
Yeah, this is a bug. I fixed it:
```
if (!wire && options.Format != ModelReaderWriterFormat.Json) throw new ArgumentOutOfRangeException(nameof(options.Format));

```